src/AppBundle/Command/AppFakerCommand.php

```diff
@@ -104,10 +104,12 @@ private function generateProduct()
     {
         $product = new Product();
         $product->setTitle($this->fk->sentence(2));
+        $product->setFacebookName($this->fk->sentence(2));
```

Comment:
use more informative variable names than "fk"... Just use "faker", it's longer, but better, because the code becomes more readable...